core/src/crds_gossip.rs

```diff
@@ -103,10 +104,10 @@ impl CrdsGossip {
 
     pub fn new_push_messages(
         &mut self,
-        pending_push_messages: Vec<(CrdsValue, u64)>,
+        pending_push_messages: Vec<CrdsValue>,
         now: u64,
     ) -> HashMap<Pubkey, Vec<CrdsValue>> {
-        self.process_push_messages(pending_push_messages);
+        self.process_push_messages(pending_push_messages, now);
```

Comment:
Will using the same "now" be a problem if there are two messages with the same key sitting in the push queue?

Reply:
I think it would be fine. The 2nd one will overwrite the 1st one since the vector is processed in order and that is consistent with how values are pushed to this queue.